### PR TITLE
Mother Anarchy loves her sons.

### PR DESCRIPTION
**Pull request by EgorDinamit**
Port from #vlggms/tegustation/pull/106

## About The Pull Request

Adds an "Anarcho-Communism Mode" admin secret to the game.
It practically makes everyone "equal" by giving them AA and similar outfits.
Also makes the economy so _freaking stable_ that everything costs much less.

## Why It's Good For The Game

https://www.youtube.com/watch?v=h_T7bD4pAlE

### CODE REVIEW

**Comment by EgorDinamit** [comment]
Damn, you don't have station traits..?

**Comment by Kitsunemitsu** [comment]
I gotta clean this up and send out again

**Comment by Kitsunemitsu** [comment]
Nice, it's updated, will get this TMed

**Review by retlaw34 — APPROVED** [review]

File: code/modules/admin/verbs/secrets.dm [hunk 1]
```diff
@@ -557,6 +557,55 @@
...
+				var/obj/item/clothing/under/newuniform = new /obj/item/clothing/under/color/black
+				if(olduniform)
+					H.temporarilyRemoveItemFromInventory(H.w_uniform, TRUE, FALSE)
+					H.equip_to_slot_or_del(newuniform, ITEM_SLOT_ICLOTHING)
+					qdel(olduniform)
+				else
+					H.equip_to_slot_or_del(newuniform, ITEM_SLOT_ICLOTHING)
+
+				// Some stuff gets yeeted.
+				qdel(H.wear_suit)
```
[the remark this answers is not in the document]
Comment (EgorDinamit) [reply code/modules/admin/verbs/secrets.dm:588]: Adding a plasmaman check.

**Review by BootlegBow — APPROVED** [review]

**Comment by BootlegBow** [comment]
alright we're good

**Pull request merged**